ruby/vampires.rb

```diff
@@ -77,6 +77,5 @@ def vampirecheck
 end 

```

Comment:
Nice work on encapsulating this into a method!

This is a solid submission on all accounts.  I only have one comment to add.

Consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.